undefined reference to `simdutf::count_utf8()` when amalgamate.py is used without `--with-latin1` argument

When using `amalgamate.py --with-utf8 --with-utf16`, linking causes an undefined reference to `simdutf::count_utf8()` (my code uses that function).

Adding `--with-latin1` works around the issue.

--- a/src/implementation.cpp
+++ b/src/implementation.cpp
@@ -2322,12 +2322,12 @@ simdutf_warn_unused size_t count_utf16be(const char16_t *input,
 }
 #endif // SIMDUTF_FEATURE_UTF16
 
-#if SIMDUTF_FEATURE_UTF8 && SIMDUTF_FEATURE_LATIN1
+#if SIMDUTF_FEATURE_UTF8
 simdutf_warn_unused size_t count_utf8(const char *input,
                                       size_t length) noexcept {
   return get_default_implementation()->count_utf8(input, length);
 }
-#endif // SIMDUTF_FEATURE_UTF8 && SIMDUTF_FEATURE_LATIN1
+#endif // SIMDUTF_FEATURE_UTF8
 
 #if SIMDUTF_FEATURE_UTF8 && SIMDUTF_FEATURE_LATIN1
 simdutf_warn_unused size_t latin1_length_from_utf8(const char *buf,

